### DISCUSSION TITLE
No possible to migrate from Trinity to Firefly??

### DISCUSSION THREAD

**Discussion by GP79MAR** [iotaledger/firefly#8090] [Help & Support]
Hi everyone,
I hope you can help me. Years ago I placed my IOTA in the Trinity Wallet and took note of the Seeding password (27 words - 81 characters). I now discover there's been a migration to the Firefly wallet and I can no longer use the 81 characters password. Is there any way around it? How can I access my funds? 
Thank you very much in advance for your support.

**Comment by begonaalvarezd** [comment]
There is a migration tool you can use https://blog.iota.org/iota-legacy-migration-tool/ please read that article for further information about it

**Comment by GP79MAR** [comment]
Thank you very much for your help!
It worked perfectly!

I really appreciate it. Thank you!
GP
